app/src/app/components/contents/downloads/Artefacts.tsx

```diff
@@ -0,0 +1,32 @@
+import { Card, CardContent, CardHeader } from "../../Base/Card";
+import FileRow from "./FileRow";
+import { Artefact, PacketMetadata } from "../../../../types";
+
+interface ArtefactProps {
+  artefacts: Artefact[];
+  packet: PacketMetadata;
+}
+
+export default function Artefacts({ artefacts, packet }: ArtefactProps) {
```

Comment:
probs best to use named exports for consistency and discoverability. 

il leave this comment but applies to all files default export used

Reply:
There are quite a few stragglers using `export default`, so I have separated this change to a new PR: https://github.com/mrc-ide/packit/pull/156